backend/src/main/scala/cromwell/backend/standard/StandardCachingActorHelper.scala

```diff
@@ -75,9 +74,7 @@ trait StandardCachingActorHelper extends JobCachingActorHelper {
 
     val fileMetadata = jobPaths.metadataPaths
 
-    val otherMetadata = Map(CallMetadataKeys.CallCaching + MetadataKey.KeySeparator + "allowResultReuse" -> true)
-
-    runtimeAttributesMetadata ++ fileMetadata ++ otherMetadata ++ nonStandardMetadata
+    runtimeAttributesMetadata ++ fileMetadata ++ nonStandardMetadata
```

Comment:
Is this going to get called a ton? presumably these maps are small but if it's going to get called a bunch in the aggregate it'd add up if `add` is linear for the default `Map` class (I'm not sure on that)